db/schema.rb

```diff
@@ -225,6 +225,8 @@
     t.datetime "expiry_time"
     t.string "supporting_documents"
     t.string "salary"
+    t.string "job_roles", array: true
```

Comment:
This should probably wait until https://github.com/DFE-Digital/teacher-vacancy-service/pull/1438 is merged

Reply:
Good spot, I will add the correct schema 👍